[SECRET-31] Add Secret Store documentation to dev-center (#452)
algorithmiaio/dev-center: _pages/platform/algorithm-secrets.md

#### Use Cases
  - Connecting to a password-protected database
  - Calling a secure HTTP webhook to `POST` your algorithm result

Secrets are encrypted at rest, encrypted in transit, and only exposed to selected algorithms. We never expose sensitive data through the web interface or any user-facing APIs.

Keep in mind that collaborators who can update algorithm source can also access its secrets.

## Managing algorithm secrets

Secrets are managed on the algorithm settings page. Here you'll be able to create, update, delete, and view your existing algorithm secrets. The steps below demonstrate the process for creating secrets.

![Image of UI settings page for creating secrets](/developers/images/post_images/algorithm_secrets/settings_page.png)

Start by clicking "New secret" to create an algorithm secret. Next, give your secret a name, description, key, and value. The key corresponds to the environment variable name and the value is the secret itself.

> It can take up to one minute for new secrets to show up inside your algorithm. Similarly, when you update a secret, it can take one minute for that change to take effect.

Reviewer: Smart addition

{: .notice-info}

![Image of UI create secret page](/developers/images/post_images/algorithm_secrets/create_secret_page.png)

After your secret is configured, it can be updated or deleted on the settings page. Clicking "Update" will direct you to a form that is identical to the form shown above, and you'll be able to change the secret's name, description, and value.

> ![Image of UI settings page for creating secrets](/developers/images/post_images/algorithm_secrets/settings_page_with_secret.png)

Reviewer: This alt tag is identical to a previous one. I'd update to something that reflects that a secret exists on the page now.
```suggestion
![Image of UI settings page with an existing secret](/developers/images/post_images/algorithm_secrets/settings_page_with_secret.png)
```
